composer.json

```diff
@@ -54,7 +54,10 @@
     "mockery/mockery": "~0.9",
     "satooshi/php-coveralls": "^1.0",
     "phpunit/phpcov": "2.*",
-    "bossa/phpspec2-expect": "~1.0"
+    "bossa/phpspec2-expect": "~1.0",
+    "puli/cli": "^1.0",
+    "elife/api": "dev-master",
+    "elife/api-validator": "dev-master"
   },
   "minimum-stability": "beta",
   "prefer-stable": true,
```

Comment:
originally @nlisgo decided not to commit `composer.lock` due to some issues with the build process where some steps would not be executed when using just `composer install` with a known `composer.lock`. Are these reasons still present?